scalikejdbc-streams/README.md

```diff
@@ -0,0 +1,54 @@
+# ScalikeJDBC Streams
+
+See the website.
+
+http://scalikejdbc.org/
+
+It is assumed to be used in batch application.
+In JDBC, you need to keep connections constantly while subscribing to streaming. Therefore, please reconsider the connection pool setting and always be careful not to exhaust the connection.
+
+
+## Example
+
+```scala
+import scalikejdbc._
+import scalikejdbc.streams._
+
+// We recommend that you prepare a ThreadPoolExecutor that generates daemon threads
+implicit val executor = AsyncExecutor(scala.concurrent.ExecutionContext.global)
+
+val publisher = DB stream {
+  sql"select id from users".map(r => r.int("id")).cursor
+}
+publisher.subscribe(???)
+```
+
+Please refer to the document on usage of ScalikeJDBC such as connection pool initialization
+http://scalikejdbc.org/documentation/configuration.html
+
+
+# Note
+
+For MySQL, PostgreSQL, and others, several settings are required to enable CURSOR (Streaming).
```

Comment:
At the current moment, scalikejdbc-streams natively supports MySQL and PostgreSQL.

For MySQL and PostgreSQL, scalikejdbc-streams automatically enables required settings to use cursor feature.